change/@microsoft-fast-tooling-4a76dab6-2d89-4cfd-a8ba-24f228cc519b.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "chore: fix tooling coverage globbing pattern",
```

```suggestion
  "comment": "fix tooling coverage globbing pattern",
```
We can probably leave "chore" off, but not necessary to change tbh...